cgi/display_org_table.pl

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/perl -w
+
+# This file is part of Product Opener.
+#
+# Product Opener
+# Copyright (C) 2011-2023 Association Open Food Facts
+# Contact: contact@openfoodfacts.org
+# Address: 21 rue des Iles, 94100 Saint-Maur des Fossés, France
+#
+# Product Opener is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use ProductOpener::PerlStandards;
+
+use CGI::Carp qw(fatalsToBrowser);
+use ProductOpener::Config qw/:all/;
+use ProductOpener::Display qw/:all/;
+use ProductOpener::Lang qw/:all/;
+use ProductOpener::Data qw/:all/;
+use Log::Any qw($log);
+
+use POSIX qw(strftime);
+
+sub format_date {
+    my ($timestamp) = @_;
+    return strftime("%Y-%m-%d %H:%M:%S", localtime($timestamp));
+}
+
+my $request_ref = ProductOpener::Display::init_request();
+
+my $orgs_collection = get_orgs_collection();
+my @orgs = $orgs_collection->find->all;
+
+foreach my $org (@orgs) {
+    $org->{created_t_readable} = format_date($org->{created_t});
```

Comment:
in the template, you can call those functions that are automatically added by process_template():

	$template_data_ref->{display_date_without_time} = \&display_date_without_time;
	$template_data_ref->{display_date_ymd} = \&display_date_ymd;
	$template_data_ref->{display_date_tag} = \&display_date_tag;